sympy/physics/vector/point.py

```diff
@@ -480,22 +507,53 @@ def vel(self, frame):
         frame : ReferenceFrame
             The frame in which the returned velocity vector will be defined in
 
+        otherpoint : Point
+            The point w.r.t which self's position is defined, works as initial
+            point.
+
```

Comment:
Remove this test, as it is no longer relevant.

Reply:
Okay sir